src/System.Text.Json/tests/Serialization/Stream.WriteTests.cs

```diff
@@ -258,6 +263,632 @@ public static async Task LargeJsonFile(int bufferSize)
+                    },
+                    References = new References
+                    {
+                        User = new User
+                        {
+                            UserId = $"7ef192b7f545",
+                            Username = "johndoe",
+                            Name = "John Doe",
+                            CreatedAt = 1456628553936,
+                            ImageId = $"{i}*h8Ph7pgNeQHZTiLlah3h-A.jpeg",
+                            BackgroundImageId = string.Empty,
+                            Bio = "I believe ideas should be open and free. This is a non-profit initiative to write about fundamentals you won't find anywhere else.",
+                            TwitterScreenName = "JohnDoe",
+                            SocialStats = new SocialStats
+                            {
+                                UserId = "7ef192b7f545",
+                                UsersFollowedByCount = 4000 + i,
+                                UsersFollowedCount = i,
+                                Type = "SocialStats"
+                            },
+                            Social = new Social
+                            {
+                                UserId = "78aeffcb483e",
+                                TargetUserId = "7ef192b7f545",
+                                Type = "Social"
+                            },
+                            FacebookAccountId = string.Empty,
+                            AllowNotes = 1,
+                            MediumMemberAt = 0,
+                            IsNsfw = false,
+                            IsWriterProgramEnrolled = true,
+                            IsQuarantined = false,
+                            Type = "User"
+                        },
+                        Social = new Social
+                        {
+                            UserId = "78aeffcb483e",
+                            TargetUserId = "7ef192b7f545",
+                            Type = "Social"
+                        },
+                        SocialStats = new SocialStats
+                        {
+                            UserId = "7ef192b7f545",
+                            UsersFollowedByCount = 4000 + i,
+                            UsersFollowedCount = i,
+                            Type = "SocialStats"
+                        }
+                    } 
+                };
+            
+                for (int j = 0; j < i % 4; j++)
+                {
+                    Paragraph purpleParagraph = new Paragraph
+                    {
+                        Name = $"a{i}b{j}",
+                        Type = i,
+                        Text = "The picture of two firefighters clearing up the snow surrounding a hydrant. In case there's an emergency, the cost in lives due to having snow blocking the hydrant is more significant than the cost of clearing them all beforehand. Clearing a hydrant after a snowfall is a problem worth solving.",
```

Comment:
We seem to be borrowing a lot of the "text" content from https://medium.com/@fagnerbrack/a-smart-programmer-understands-the-problems-worth-fixing-dcf15871f943

Can you just use dummy data instead? I think we should also be cautious when including anything that came specifically from the article (including links, image ids, etc.).

Reply:
I recommend [Project Gutenberg](http://www.gutenberg.org/) for large bodies of text.

Reply:
Sure, links, names, images, ids updated/dummied, texts seemed to me dummy enough, but will update too

Reply:
The guidance is simple: don't take content from an unapproved resource. The whole of the web is that.

I've pre-approved two resources for the team and we can add others. For clarity, for content like this, anything that is public domain or has an appropriate creative commons license is fine. You'll have to show me the CC license if you want to use CC content.

Reply:
How about adding this comment:
`// This schema populated using blogpost structure of https://medium.com`
https://github.com/dotnet/corefx/pull/39668/files#diff-d488e84455a3af1d990d5343131ade08R10
Any application talking to the site would need to produce json structure using their payload, so i assume its OK, used only schema and all data from payload dummied.

Reply:
Sure. It is fine to use an API for a commercial service, like medium or Netflix. It is a problem if we store user content (like blog posts, in part or in whole) in tests because it is testing what the service returns. We may or may not  have to the rights to that, and I'm not interested in finding out in most cases (because it's a losing proposition, and there are better options).

I assume we're not actually going to call the medium service as part of our test runs. Is that right?

Reply:
> I assume we're not actually going to call the medium service as part of our test runs. Is that right?

Right.